conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conans import ConanFile, CMake, tools
+
+
+class GoogleBenchmarkConan(ConanFile):
+    name = "benchmark"
+    version = "1.4.1"
+    description = "A microbenchmark support library."
+    url = "https://github.com/google/benchmark"
+    homepage = "https://github.com/google/benchmark"
+    license = "Apache-2.0"
+    settings = "arch", "build_type", "compiler", "os"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "enable_exceptions": [True, False],
+        "enable_lto": [True, False],
+        "enable_testing": [True, False],
+        "enable_gtest_tests": [True, False]
+    }
+    default_options = "shared=False", "fPIC=True", "enable_exceptions=True", "enable_lto=False", "enable_testing=False", "enable_gtest_tests=False"
+    exports_sources = ["*"]
+    generators = "cmake"
+
+    build_subfolder = "."
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.settings.os == 'Windows' and self.options.shared:
+            raise Exception("Windows shared builds are not supported right now, see issue #639")
+
+        if self.options.enable_testing == False:
+            self.options.enable_gtest_tests = False
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions['BENCHMARK_ENABLE_TESTING'] = "ON" if self.options.enable_testing else "OFF"
```

Comment:
if this is enabled, i think you'd also want BENCHMARK_DOWNLOAD_DEPENDENCIES to be ON, or the build won't find googletest.

Reply:
Ah, just saw the build_requires below. nice.